File: mobile/src/main/java/com/example/floriel/orangesafetyservices/fragments/SettingsFragment.java
package com.example.floriel.orangesafetyservices.fragments;

import android.os.Bundle;
import android.support.v7.preference.PreferenceFragmentCompat;

import com.example.floriel.orangesafetyservices.R;

public class SettingsFragment extends PreferenceFragmentCompat {

    public SettingsFragment() {
        // Required empty public constructor
    }

    public static SettingsFragment newInstance(int instance) {
        SettingsFragment fragment = new SettingsFragment();
        Bundle args = new Bundle();
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.preferences);
    }

}

File: mobile/src/main/java/com/example/floriel/orangesafetyservices/feature/settings/SettingsFragment.kt
package com.example.floriel.orangesafetyservices.feature.settings

import android.os.Bundle
import android.support.v7.preference.PreferenceFragmentCompat
import com.example.floriel.orangesafetyservices.R

class SettingsFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        addPreferencesFromResource(R.xml.preferences)
    }

    companion object {

        fun newInstance(): SettingsFragment {
            return SettingsFragment()
        }
    }

}
